src/slic3r/Utils/PresetUpdater.cpp

```diff
@@ -1079,9 +1079,9 @@ void PresetUpdater::priv::check_installed_vendor_profiles() const
     AppConfig *app_config = GUI::wxGetApp().app_config;
     const auto enabled_vendors = app_config->vendors();
 
-    std::vector<std::string> bundles;
+    std::set<std::string> bundles;
```

Comment:
We need to make sure ORCA_FILAMENT_LIBRARY is loaded first, so that other profiles can inherit from it.
So we have to use vector here to make sure the insertion order is respected